ui/src/tree/AzExtTreeDataProvider.ts

```diff
@@ -23,6 +23,7 @@ export class AzExtTreeDataProvider implements IAzExtTreeDataProviderInternal, ty
 
     private readonly _loadMoreCommandId: string;
     private readonly _rootTreeItem: AzExtParentTreeItem;
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
```

Comment:
What's causing this to be flagged??

Reply:
It didn't like that `new Map()` didn't have a typecast. I added one and removed this exception